src/cascadia/TerminalApp/App.idl

```diff
@@ -24,6 +25,7 @@ namespace TerminalApp
         Boolean GetShowTabsInTitlebar();
 
         event Microsoft.Terminal.TerminalControl.TitleChangedEventArgs TitleChanged;
+        event LastTabClosedEventArgs CloseWindow;

```

Comment:
It looks like the event is still called CloseWindow

Reply:
Oops, yeah missed it, sorry.

Reply:
Hopefully it's done this time, misunderstood what you meant earlier and thought that it was only about EventArgs...